packages/tools/examples/ultrasoundAnnotation/index.ts

```diff
@@ -0,0 +1,277 @@
+  await initDemo();
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(StackScrollTool);
+  cornerstoneTools.addTool(ZoomTool);
+  cornerstoneTools.addTool(UltrasoundAnnotationTool);
+  cornerstoneTools.addTool(WindowLevelTool);
+
+  // Define a tool group, which defines how mouse events map to tool commands for
+  // Any viewport using the group
+  const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+
+  // Add tools to the tool group
+  toolGroup.addTool(WindowLevelTool.toolName);
+  toolGroup.addTool(ZoomTool.toolName);
+  toolGroup.addTool(StackScrollTool.toolName, { loop: false });
+  toolGroup.addTool(UltrasoundAnnotationTool.toolName);
+
+  // Set the initial state of the tools, here all tools are active and bound to
+  // Different mouse inputs
+  toolGroup.setToolActive(defaultLeftClickTool, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Left Click
+      },
+    ],
+  });
+  toolGroup.setToolActive(WindowLevelTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Auxiliary, // Middle Click
+      },
+    ],
+  });
+  toolGroup.setToolActive(ZoomTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Secondary, // Right Click
+      },
+    ],
+  });
+  // As the Stack Scroll mouse wheel is a tool using the `mouseWheelCallback`
+  // hook instead of mouse buttons, it does not need to assign any mouse button.
+  toolGroup.setToolActive(StackScrollTool.toolName, {
+    bindings: [{ mouseButton: MouseBindings.Wheel }],
+  });
+
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const imageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.1188.2803.137585363493444318569098508293',
+    SeriesInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.1188.2803.699272945123913604672897602509',
+    SOPInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.1188.2803.260509337872681089220763036630',
+    wadoRsRoot: 'https://d14fa38qiwhyfd.cloudfront.net/dicomweb',
+  });
+
+  // Instantiate a rendering engine
+  const renderingEngineId = 'myRenderingEngine';
+  renderingEngine = new RenderingEngine(renderingEngineId);
+
+  // Create a stack viewport
+
+  const viewportInput = {
+    viewportId,
+    type: ViewportType.STACK,
+    element,
+    defaultOptions: {
+      background: <Types.Point3>[0.2, 0, 0.2],
+    },
+  };
+
+  renderingEngine.enableElement(viewportInput);
+
+  // Set the tool group on the viewport
+  toolGroup.addViewport(viewportId, renderingEngineId);
+
+  // Get the stack viewport that was created
+  const viewport = <Types.IStackViewport>(
+    renderingEngine.getViewport(viewportId)
+  );
+
+  // Set the stack on the viewport
+  viewport.setStack(imageIds);
+
+  toolGroup.setToolConfiguration(UltrasoundAnnotationTool.toolName, {
+    fanCenter: [440, -270, 0],
+    innerRadius: 220.6,
+    outerRadius: 682.8,
+    startAngle: 65,
+    endAngle: 115,
+  });
```

Comment:
I think we should implement it using metadata providers. So have a specific metadata provider that provides this given the imageId, write a custom metadataprovider and have these hardcoded for now, and inside the tool we look for metadata.get('ultrasoundFanShapeWhatever', imagId)